scripts/create-partner-chart

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+set -e
+
+usage="Usage: make create URL={REQUIRED_URL_TO_CHART} CHART={OPTIONAL_CHART_DIR_NAME}"
+
+# Verify required arguments
+if [[ ! $URL ]]; then
+    echo "Error: Missing required argument URL"
+    echo $usage
+    exit 1
+fi
+
+# Verify URL is valid and references a tgz resource
+if [[ ! $URL =~ https?://.*\.tgz ]]; then
+    echo "Error: Invalid URL form"
```

Comment:
Should specify that its to a tgz, they might put index.yaml

Reply:
I'll update it to include that.